README.md

```diff
@@ -14,30 +14,69 @@ an exact solution if possible or use an appropriate numeric solver otherwise.
 
 `src` - The source code of NESTML
 
-## Docker
-The docker files can be find in the `docker` folder.
+## Installing and running NESTML
 
-For the usage:
-Docker must be installed on the machine. See the follwing resource for the installation instructions https://docs.docker.com/engine/installation
-Clone the github repository locally. Switch to the docker-folder in the nestml repository.
+To install NESTML, the following requirements need to be met:
+* [Apache Maven](https://maven.apache.org/)
```

Comment:
Maybe mention available directly in Ubuntu/Debian through ``sudo apt-get install maven`` or in Arch (``pacman -S maven``)